server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -449,6 +447,13 @@ public static void main(String[] args) throws Exception {
       log.info("SASL is not enabled, delegation tokens will not be available");
       delegationTokensAvailable = false;
     }
+    long waitTimeBetweenScans =
+        aconf.getTimeInMillis(Property.MANAGER_TABLET_GROUP_WATCHER_INTERVAL);
+    this.timeToCacheRecoveryWalExistence = waitTimeBetweenScans / 4;
+  }
+
+  public long getWaitTimeBetweenScans() {
+    return this.getConfiguration().getTimeInMillis(Property.MANAGER_TABLET_GROUP_WATCHER_INTERVAL);
```

Comment:
The `getWaitTimeBetweenScans()` method returns the current value, but `this.timeToCacheRecoveryWalExistence` is still based on, and fixed to, the initial value. It doesn't seem like a good idea to have a property be runtime-mutable, but also be the basis for a runtime-immutable characteristic, at the same time. Should these be separate properties, one that is a fixedProperty and one that is mutable at runtime?

Reply:
Created a separate property in ff6923e